### PR TITLE
Introduce monorepo

### PR DESCRIPTION
**Pull request by marekkirejczyk**
Closes https://github.com/EthWorks/useDapp/issues/16

### CODE REVIEW

**Comment by rzadp** [comment]
@marekkirejczyk There are 2 `yarn.lock`'s - by mistake or on purpose?

File: package.json [hunk 1]
```diff
@@ -1,45 +1,23 @@
...
+      "build": "wsrun -e -c -s build",
+      "test": "wsrun -e -c -s --exclude-missing test"
+    },
+    "dependencies": {
+      "wsrun": "^5.2.1",
+      "prettier": "2.1.2"
+    }
   }
-}
+
```
Comment (rzadp) [inline package.json:23]:
```suggestion

```

File: package.json [hunk 2]
```diff
@@ -1,45 +1,22 @@
...
+    "scripts": {
+      "lint": "wsrun -c -s lint",
+      "build": "wsrun -e -c -s build",
+      "test": "wsrun -e -c -s --exclude-missing test"
+    },
+    "dependencies": {
+      "wsrun": "^5.2.1",
+      "prettier": "2.1.2"
+    }
   }
```
Comment (rzadp) [inline package.json:22]:
```suggestion
}
```

File: packages/core/package.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@usedapp/core",
+  "version": "1.0.0",
+  "main": "src/index.ts",
+  "repository": "git@github.com:EthWorks/dapp-framework.git",
+  "author": "Justyna <jusbroniszewska@gmail.com>",
```
Comment (rzadp) [inline packages/core/package.json:6]:
```suggestion
  "author": "Ethworks",
```

@JustynaBroniszewska If you're OK with this :D

Comment (JustynaBroniszewska) [reply packages/core/package.json:6]: hah, of course 😄

File: packages/core/package.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    "react-dom": "^17.0.1",
+    "styled-components": "^5.2.1"
+  },
+  "scripts": {
+    "start": "webpack serve --mode development",
+    "build": "echo Nothing to build",
+    "test": "mocha",
+    "lint": "yarn lint:prettier --check && yarn lint:eslint",
+    "lint:fix": "yarn lint:prettier --write && yarn lint:eslint --fix",
+    "lint:eslint": "eslint './test/**/*.{ts,tsx}'",
```
Comment (rzadp) [inline packages/core/package.json:43]:
```suggestion
    "lint:eslint": "eslint './{src, test}/**/*.{ts,tsx}'",
```

**Review by rzadp — APPROVED** [review]

**Pull request merged**